### PR TITLE
New exportArchive Feature

### PR DESCRIPTION
**Pull request by KrauseFx**
Follow up on https://github.com/fastlane/gym/pull/79

### CODE REVIEW

File: lib/gym/options.rb [hunk 1]
```diff
@@ -101,6 +101,16 @@ def self.plain_options
                                      verify_block: proc do |value|
                                        raise "File not found at path '#{File.expand_path(value)}'".red unless File.exist?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :skip_symbols,
+                                     short_option: "-m",
+                                     env_name: "GYM_SKIP_SYMBOLS",
+                                     description: "Should the ipa file *not* include symbols?",
```
Comment (ldiqual) [inline lib/gym/options.rb]: Any reason for using a negation in the flag name here? Seems confusing, especially when the actual archive flag (`uploadSymbols`) doesn't have a negation. How about `GYM_UPLOAD_SYMBOLS` with default `false`?

Comment (KrauseFx) [inline lib/gym/options.rb]: Very good point, thank you so much @ldiqual :+1:

Comment (KrauseFx) [inline lib/gym/options.rb]: The reason for that was, that you can't pass a `false` value from the command line right now.

**Comment by ldiqual** [comment]
This looks really good, I'm excited to see this in master! I believe the new archiver will need to use this hack to support swift apps: http://stackoverflow.com/a/32535087/646960. Without it, I wasn't able to submit my binaries because of a validation error (the infamous `Invalid swift support`). I added this simple bash command in my script:

```
build_archive() # xcodebuild archive

cp "${TMP_ARCHIVE_PATH}.xcarchive/SwiftSupport/iphoneos"/libswift*.dylib \
    "${TMP_ARCHIVE_PATH}.xcarchive/Products/Applications/${PRODUCT_NAME}.app/Frameworks/"

generate_ipa() # New -exportArchive command
```

Let me know if you need some extra hands, I'll be happy to help.

**Comment by lacostej** [comment]
+1 thanks @KrauseFx

**Comment by KrauseFx** [comment]
Yeah, thanks @lacostej for preparing everything :+1:

**Pull request merged**